[2024-05-24T03:48:52.371+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:48:50.994887+00:00 [queued]>
[2024-05-24T03:48:52.383+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:48:50.994887+00:00 [queued]>
[2024-05-24T03:48:52.384+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-05-24T03:48:52.397+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): execute_teste_py> on 2024-05-24 03:48:50.994887+00:00
[2024-05-24T03:48:52.405+0000] {standard_task_runner.py:57} INFO - Started process 530 to run task
[2024-05-24T03:48:52.408+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'minio_python_task', 'execute_teste_py', 'manual__2024-05-24T03:48:50.994887+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dag_teste/teste_***.py', '--cfg-path', '/tmp/tmpibuh5u3n']
[2024-05-24T03:48:52.410+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask execute_teste_py
[2024-05-24T03:48:52.492+0000] {task_command.py:415} INFO - Running <TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:48:50.994887+00:00 [running]> on host 996d4eb998ba
[2024-05-24T03:48:52.624+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_python_task' AIRFLOW_CTX_TASK_ID='execute_teste_py' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T03:48:50.994887+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-24T03:48:50.994887+00:00'
[2024-05-24T03:48:52.627+0000] {logging_mixin.py:151} INFO - celery
[2024-05-24T03:48:52.627+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-24T03:48:52.640+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=minio_python_task, task_id=execute_teste_py, execution_date=20240524T034850, start_date=20240524T034852, end_date=20240524T034852
[2024-05-24T03:48:52.668+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-24T03:48:52.694+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
